src/main/resources/assets/computercraft/lua/rom/programs/size

```diff
@@ -0,0 +1,27 @@
+local Arg = {...}
+
+if Arg[1] == nil then
+  print("Usage: size <path>")
+  return 1
+end
+
+local filename = shell.resolve(Arg[1])
+
+if fs.exists(filename) == true then
+  if fs.isDir(filename) then
+    os.loadAPI("/usr/apis/wilmaapi")
+    local fileta = wilmaapi.listAllFiles(filename)
```

Comment:
Preferably load your API into the local scope. Even more preferably, don't use a non-CC api at all!